qpc/request.py

```diff
@@ -26,6 +27,43 @@
 DELETE = 'DELETE'
 PUT = 'PUT'
 
+CONNECTION_ERROR_MSG = messages.CONNECTION_ERROR_MSG
+SSL_ERROR_MSG = messages.SSL_ERROR_MSG
+
+# pylint: disable=invalid-name
+try:
+    exception_class = json.decoder.JSONDecodeError
+except AttributeError:
+    exception_class = ValueError
+
+
+def handle_token_errors(response):
+    """Handle token errors.
+
+    :param response: The response object.
+    :returns: The response object.
+    """
+    token_expired = {'detail': 'Token has expired'}
```

Comment:
Will 'Token has expired' ever be translated?

Reply:
No, third party tool doesn't.
https://github.com/JamesRitchie/django-rest-framework-expiring-tokens/blob/master/rest_framework_expiring_authtoken/authentication.py#L34

Reply:
👍